src/components/KStepper/KStep.vue

```diff
@@ -37,9 +37,9 @@ const props = defineProps({
     required: true,
   },
   state: {
-    type: String as PropType<StepperState>,
+    type: String as PropType<StepperState | ''>,
```

Comment:
If we have `''` empty string as an element in the `StepperStateArray` then we wouldn't have to specify it as the fallback here.

Reply:
removed